.github/workflows/march-image-build-push.yml

```
@@ -16,7 +16,7 @@ concurrency:
 jobs:
   push-quay:
     name: Build and Push Manifest
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

```suggestion
    runs-on: ubuntu-latest
```
https://docs.github.com/en/actions/writing-workflows/choosing-where-your-workflow-runs/choosing-the-runner-for-a-job#standard-github-hosted-runners-for-public-repositories